vendor/engines/secure_rooms/app/views/secure_rooms/facility_occupancies/index.html.haml

```diff
@@ -0,0 +1,5 @@
+= content_for :h1 do
+  =h current_facility
```

Comment:
I know some of the old partials have the `h` from old Rails, but we should get rid of it where we can.